packages/core/src/repository/query/network-query.ts

```diff
@@ -0,0 +1,20 @@
+import { ParameterType } from '../../data';
+
+export abstract class NetworkQuery {
+    constructor(public readonly endpoint: string) {}
```

Comment:
@orioljp Question: the endpoint includes the domain or only the path?.

Reply:
It should be only the path. See packages/core/src/repository/data-source/api-request.service.ts , builder method. The query.endpoint is being used when calling the method. We could change the parameter's name if it makes it clearer.

Reply:
@orioljp @lucianosantana and this behaviour allow us to point to two APIs with different domains?.

Reply:
Good point @Alex-DA . The base URL is defined on the `ApiRequestService`. In this case, we'd have to create a new service instance and a new NetworkDataSource for each API base URL. So the setting is done via dependency injection. We could change it to have the base URL defined on every query. Both are a bit of extra work. Having it on the query seems to be more work for me.

Reply:
@Alex-DA Yes, we can have many base urls but not in the same datasource, since a datasource is one, two base urls would mean two sources of data, and this is managed by providing 2 different ApiRequestServices with different urls. Anyway, the behaviour of the baseUrl managing has not changed in this PR

Reply:
Maybe we can change this from `endpoint` to `path`? Or, we can add a docblock for the `endpoint` var explaining that the `baseUrl` is managed elsewhere.

Reply:
changed to path